synapse/lib/stormtypes.py

```diff
@@ -52,13 +52,13 @@ class StormTypesRegistry:
     base_undefined_types = (
         'any',
         'int',
```

Comment:
We should sort out `int` and `integer` overloading here. That seems like an error? I will double check it.

Reply:
It looks like we use `integer` in 2 places ( the lib.range and imap server port ), and `int` primarily. I'd be in favor of removing `integer` and cleaning those instances up while we are in here?

Reply:
sounds good, i'll clean that up